src/libraries/Microsoft.PowerFx.Core/Utils/MutationUtils.cs

```diff
@@ -65,5 +70,44 @@ public static void CheckSemantics(TexlBinding binding, TexlFunction function, Te
                 return;
             }
         }
+
+        public static string GetScalarSingleColumnNameForType(Features features, DKind kind)
+        {
+            return kind switch
+            {
+                DKind.Image or
+                DKind.Hyperlink or
+                DKind.Media or
+                DKind.Blob or
+                DKind.PenImage => features.ConsistentOneColumnTableResult ? TableValue.ValueName : "Url",
+
+                _ => TableValue.ValueName
+            };
+        }
+
+        public static List<IntermediateNode> CreateIRCallNodeCollect(CallNode node, IRTranslator.IRTranslatorContext context, List<IntermediateNode> args, ScopeSymbol scope)
```

Comment:
>CreateIRCallNodeCollect [](http://example.com/codeflow?start=45&length=23)

Same question as above - since this is only used by CollectFunction, can it live there?